src/pages/Transactions/Transaction.tsx

```diff
@@ -382,8 +385,6 @@ function renderStateCheckpointTransaction(
           {renderRow("Version:", transaction.version)}
           {renderRow("Hash:", transaction.hash)}
           {renderRow("Status:", renderSuccess(transaction.success))}
-          {"proposer" in transaction &&
```

Comment:
Did it get renamed or is it meant to be removed entirely?

Reply:
was removed